### PR TITLE
Travis CI setup

### PR DESCRIPTION
**Pull request by rafaelfranca**
Here is an example build https://travis-ci.org/rafaelfranca/shipit-engine/builds/64195217.

In my account the jobs started in less than 10 seconds, so maybe the queue size will be not a problem (but I don't know if this is because people on Travis gave me some special treatment since I was working with them beta testing the new container setup with Rails).

There is still a failure because a problem with datetime comparing on sqlite that I'm still investigating.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'mysql2'
+gem 'pg'
+gem 'sqlite3'
```
Comment (gmalette) [inline Gemfile]: Can anyone explain to me why this is in the `Gemfile` and not in the `gemspec`? I'm not sure I understand which is for what

**Comment by gmalette** [comment]
I think this is much nicer. If queues are an issue we can go back

File: Gemfile (hunk 1 above)
Comment (byroot) [inline Gemfile]: Gemspec are them gem dependencies. While  in the gemfile you put things that are only for dev/test

**Comment by byroot** [comment]
LGTM. Feel free to mark sqlite as soft-fail and we can fix it in another PR.

File: Gemfile (hunk 1 above)
Comment (rafaelfranca) [inline Gemfile]: You can put development dependencies at gemspec too but you will loose the ability to define bundle groups. The only advantage of having development dependencies is that they will be present on the rubygems pages like this https://rubygems.org/gems/arel.

Comment (gmalette) [inline Gemfile]: :+1: thanks that clarifies it

**Comment by rafaelfranca** [comment]
Done. I also added `db:seed` to the test script.

Now you just need to enable travis builds.

**Comment by byroot** [comment]
I was able to tell you to merge yourself, I guess I'm not used to external contributors yet :p.

**Comment by byroot** [comment]
Builds are now enabled in travis: https://travis-ci.org/Shopify/shipit-engine/branches

File: script/cibuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# Output all commands, stop on failures
+set -e
+set -x
+
+cp -n test/dummy/config/secrets.example.yml test/dummy/config/secrets.yml || true
+
+case $DATABASE in
+  mysql)
+    echo "Testing PostgreSQL"
```
Comment (arthurnn) [inline script/cibuild]: Shouldnt say mysql here?

Comment (rafaelfranca) [inline script/cibuild]: Yup

**Pull request merged**